hare/hare_test.go

```diff
@@ -243,6 +251,11 @@ func TestHare_OutputCollectionLoop(t *testing.T) {
 	h.broker.mu.RUnlock()
 }
 
+func proposalsDiff(t *testing.T, pList, got []*types.Proposal) string {
+	less := func(a, b *types.Proposal) bool { return a.ID().String() < b.ID().String() }
+	return cmp.Diff(pList, got, cmpopts.EquateEmpty(), cmp.AllowUnexported(types.Proposal{}, types.Ballot{}, signing.PublicKey{}), cmpopts.SortSlices(less))
```

Comment:
require.Empty(tb, ...)

Reply:
`proposalsDiff` has been removed after merging develop as not used anywhere.